cloudbuild_app_dart.yaml

```diff
@@ -1,6 +1,6 @@
 # Provide instructions for google Cloud Build to auto-build flutter
 # dashboard to flutter-dashboard project. Auto-build will be triggered
```

Comment:
Are we able to move these yamls into the app_dart and auto_submit directories?

Reply:
Let me play around.

Reply:
Now this works.